module/zfs/trace.c

```diff
@@ -18,8 +18,11 @@
  *
  * CDDL HEADER END
  */
+
 /*
- * Each Linux tracepoints subsystem must define CREATE_TRACE_POINTS in one
+ * Each DTRACE_PROBE must define its trace point in one (and only one)
+ * source file, so this dummy file exists for that purpose.  The standard
+ * Linux tracepoints required that CREATE_TRACE_POINTS be defined in one
  * (and only one) C file, so this dummy file exists for that purpose.
```

Comment:
Probably my fault from the original prototype, but the phrasing here is a bit redundant.  How about just:

```
 * Each DTRACE_PROBE must define its trace point in one (and only one)
 * source file, so this dummy file exists for that purpose.
```